zerver/tests/test_message_fetch.py

```diff
@@ -527,7 +535,7 @@ def test_build_narrow_filter(self) -> None:
         fixtures_path = os.path.join(os.path.dirname(__file__), "fixtures/narrow.json")
         with open(fixtures_path, "rb") as f:
             scenarios = orjson.loads(f.read())
-        self.assertTrue(len(scenarios) == 9)
+        self.assert_length(scenarios, 10)
```

Comment:
I'm adding a prep commit for: `assertTrue` -> `assert_length`. (https://github.com/akshatdalton/zulip/commit/45d34d75a0a21a6d12572dac8aa2077c4700ca73)